app/services/subscriptions/create_service.rb

```diff
@@ -2,7 +2,8 @@
 
 module Subscriptions
   class CreateService < BaseService
-    attr_reader :current_customer, :current_plan, :current_subscription, :name, :external_id, :billing_time
+    attr_reader :current_customer, :current_plan, :current_subscription, :name, :external_id, :billing_time,
```

Comment:
Layout/MultilineMethodArgumentLineBreaks: Each argument in a multi-line method call must start on a separate line.